feature-libs/cart/quick-order/components/quick-order/quick-order.component.ts

```diff
@@ -9,21 +14,26 @@ import {
   OrderEntry,
 } from '@spartacus/core';
 import { CmsComponentData } from '@spartacus/storefront';
-import { Observable } from 'rxjs';
+import { combineLatest, Observable, Subscription } from 'rxjs';
 import { filter, first, map, switchMap } from 'rxjs/operators';
 
 @Component({
   selector: 'cx-quick-order',
   templateUrl: './quick-order.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class QuickOrderComponent implements OnInit {
+export class QuickOrderComponent implements OnInit, OnDestroy {
   cartId$: Observable<string>;
   entries$: Observable<OrderEntry[]>;
-  isCartLoading$: Observable<boolean> = this.activeCartService.isStable();
   quickOrderListLimit$: Observable<
     number | undefined
   > = this.component.data$.pipe(map((data) => data.quickOrderListLimit));
+  isCartStable$: Observable<any> = combineLatest([
```

Comment:
Let's talk after/before standup

I'm not quite sure why we need a spinner for your page as it's not dependent on any data especially with cart.